services/validators_test.go

```diff
@@ -152,6 +153,9 @@ func TestValidateServiceAgreement(t *testing.T) {
 
 	basic := cltest.EasyJSONFromFixture("../internal/fixtures/web/hello_world_agreement.json")
 	basic = basic.Add("oracles", oracles)
+	// XXX: Add this once we find out how to adjust the config file
+	// threeDays, _ := time.ParseDuration("72h")
+	// basic = basic.Add("endAt", time.Now().Add(threeDays))
```

Comment:
We try not to leave commented out code when we merge to master. Should we make an issue for this in pivotal tracker?

Reply:
I resurrected this code in commit 84308154. The tests pass without further modification.

If you meant an issue for automatically detecting commented-out code, that sounds potentially useful, if it's easy to do.